muffnn/mlp/mlp_classifier.py

```diff
@@ -53,6 +53,13 @@ class MLPClassifier(MLPBaseEstimator, ClassifierMixin):
         If int, the random number generator seed. If RandomState instance,
         the random number generator itself. If None, then `np.random` will be
         used.
+    solver: an instance of a subclass of `tf.train.Optimizer`, optional
+        The solver to use to minimize the loss. The default is
+        `tf.train.AdamOptimizer`.
```

Comment:
I mean yes, but I hate having to both remember the function signature and read the docs to figure out what is set by default. I will remove anyways.